packages/sysutils/systemd/package.mk

```diff
@@ -187,7 +187,12 @@ post_makeinstall_target() {
   sed -e "s,^.*HandleLidSwitch=.*$,HandleLidSwitch=ignore,g" -i $INSTALL/etc/systemd/logind.conf
   sed -e "s,^.*HandlePowerKey=.*$,HandlePowerKey=ignore,g" -i $INSTALL/etc/systemd/logind.conf
 
-  # replace systemd-machine-id-setup with ours
+  # fix ordering cycle for overlay mounts
+  if [ "$DISTRO" = "Lakka" ]; then
```

Comment:
Do we really care about breaking the LibreELEC distro? I think this condition could be dropped

Reply:
It is easier for resolving conflicts (when merging libreelec) if we know why it was added and that it is for Lakka. I did other changes to upstream files like this too, i.e.:
https://github.com/libretro/Lakka-LibreELEC/blob/4a39bb14245dec24746bdc0223798587a74a3db4/packages/graphics/libdrm/package.mk#L56-L57
or
https://github.com/libretro/Lakka-LibreELEC/blob/596ea200359c0896f5c531aa7348c790ca5adefa/packages/graphics/mesa/package.mk#L61-L64

Reply:
I feel like a comment does the same, like `### LAKKA ###` or whatever thing we can easily grep

Reply:
Yes, a comment would do, but I think we should care about not breaking LE, if it is possible. A one condition extra does not cost extra maintenance. It is extra "code", even if it would be just one line, or one line in a condition. Maybe in future we will need do more changes to the systemd units and this condition would come handy - "oh, the changes can go here" ;-) Again, comment would do the same job, but would break LE and that is not what my OCD allows me to do :-D

Reply:
Fair enough, I find it a little heavy to add the condition to all the packages we edit but if it's how you want to do I don't mind